src/docs/release/breaking-changes/rendereditable-stop-painting-invalid-selection.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Change RenderEditable to stop painting the caret when its selection is invalid
+description: RenderEditable no longer paints the caret when its selection is (-1, -1).
```

Comment:
Maybe we should say "when its selection is invalid", we also no longer paint when it is (-2,-2).

Reply:
It can't go below -1 but yeah you're right anything negative makes it invalid.